src/current/cockroachcloud/learn-cockroachdb-sql.md

```diff
@@ -8,12 +8,12 @@ docs_area: get_started
 This tutorial walks you through some of the most important CockroachDB SQL statements. For a list of all supported SQL statements, see [SQL Statements](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/sql-statements).
 
 {{site.data.alerts.callout_info}}
-This tutorial is for {{site.data.products.dedicated}} or {{site.data.products.serverless}} users.
+This tutorial is for {{ site.data.products.cloud }} users.
```

Comment:
Weird wording, we may want to consider changing it to something like "This tutorial requires a cluster deployed on {{ site.data.products.cloud }}" or something like tha for these. It is more about where the cluster is deployed than where the users are, right?.